<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ManufacturingSetupScreen</class>
 <widget class="QDialog" name="ManufacturingSetupScreen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>272</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(49, 49, 49);</string>
  </property>
  <widget class="QPushButton" name="backManufacturingSetup">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>31</y>
     <width>38</width>
     <height>30</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">border:none;
border-radius:3px;
background-color: rgb(70, 70, 73);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="../rsc/resource.qrc">
     <normaloff>:/icons/previous.png</normaloff>:/icons/previous.png</iconset>
   </property>
  </widget>
  <widget class="QLabel" name="titleManufacturingSetup">
   <property name="geometry">
    <rect>
     <x>83</x>
     <y>36</y>
     <width>231</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Roboto</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:#1788c3
</string>
   </property>
   <property name="text">
    <string>Manufacturing Setup</string>
   </property>
  </widget>
  <widget class="QScrollArea" name="scrollAreaManufacturingSetup">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>70</y>
     <width>480</width>
     <height>200</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"> QScrollBar:vertical {
     border: none;
     width: 5px;
 }
 QScrollBar::handle:vertical {
     background:#1788c3;
     min-height: 20px;
 }
 QScrollBar::up-arrow:vertical, QScrollBar::down-arrow:vertical {
     border: 2px #1788c3;
     width: 5px;
     height: 5px;
     background: #1788c3;
 }</string>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOn</enum>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContentsManufacturingSetup">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>464</width>
      <height>198</height>
     </rect>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>9</y>
       <width>461</width>
       <height>201</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayoutManufacturingSetup">
      <property name="leftMargin">
       <number>60</number>
      </property>
      <property name="topMargin">
       <number>5</number>
      </property>
      <property name="rightMargin">
       <number>60</number>
      </property>
      <property name="bottomMargin">
       <number>5</number>
      </property>
     </layout>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="../rsc/resource.qrc"/>
 </resources>
 <connections/>
</ui>
